include/sdf/Sensor.hh

```diff
@@ -65,8 +66,8 @@ namespace sdf
     /// \brief A force-torque sensor.
     FORCE_TORQUE = 5,
 
-    /// \brief A GPS sensor.
-    GPS = 6,
+    /// \brief A SatNav sensor, such as GPS.
+    SATNAV = 6,
```

Comment:
We should keep GPS here for a while for backward compatibility. Could you add a new enum at the end with the number 21? Keeping GPS and moving this to the end should make sure we don't break ABI.

Reply:
agreed; here's a link to the ABI checker report:

* [![Build Status](https://build.osrfoundation.org/buildStatus/icon?job=sdformat-abichecker-any_to_any-ubuntu_auto-amd64&build=1093)](https://build.osrfoundation.org/job/sdformat-abichecker-any_to_any-ubuntu_auto-amd64/1093/) https://build.osrfoundation.org/job/sdformat-abichecker-any_to_any-ubuntu_auto-amd64/1093/API_5fABI_20report/

Reply:
Should be fixed now in https://github.com/osrf/sdformat/pull/453/commits/b128c9cc8c5aad02e8db480bd25626ae8769c545. Let's see what the ABI checker says :crossed_fingers:

Reply:
ABI checker is green, so I'm marking this as resolved